core/feature_flag_list.py

```diff
@@ -96,9 +95,8 @@ class FeatureNames(enum.Enum):
     FeatureNames.CD_ADMIN_DASHBOARD_NEW_UI,
     FeatureNames.SERIAL_CHAPTER_LAUNCH_CURRICULUM_ADMIN_VIEW,
     FeatureNames.DIAGNOSTIC_TEST,
-    FeatureNames.ENABLE_VOICEOVER_CONTRIBUTION,
     FeatureNames.SERIAL_CHAPTER_LAUNCH_LEARNER_VIEW,
-    FeatureNames.AUTO_UPDATE_EXP_VOICE_ARTIST_LINK
+    FeatureNames.ADD_VOICEOVER_WITH_ACCENT,
```

Comment:
The functionality to add/access voiceovers with accents is approved in this PR (https://github.com/oppia/oppia/pull/20163). Now moving this flag to TEST in order to verify the functionality in test server. Thanks